dataset/dictionary/characters.json5

```diff
@@ -3587,12 +3576,32 @@
     tags: [ "sumeru", "character-sub" ],
     notes: "甘露の池の前にいる花霊",
   },
+  {
+    en: "Aarav",
+    ja: "アーラヴ",
+    zhCN: "阿拉夫",
+    tags: [ "sumeru", "character-sub" ],
+    notes: "マハマトラの1人",
+  },
+  {
+    en: "Bamoun",
+    ja: "バムーン",
+    zhCN: "巴穆恩",
+    notes: "セノの伝説任務に登場する人物",
+    tags: [ "sumeru", "character-sub" ],
+  },
+  {
+    en: "Naphis",
+    ja: "ナフィス",
+    zhCN: "纳菲斯",
+    tags: [ "sumeru", "character-sub" ],
+  },
   {
     en: "Atossa",
     ja: "アトッサ",
     zhCN: "阿托莎",
     tags: [ "sumeru", "character-sub" ],
-    notes: "魔神任務 第四章第六幕「ベッドタイムストーリー」に登場するキャラクター",
+    notes: "魔神任務 第四章第六幕「ベッドタイムストーリー」に登場する人物",
```

Comment:
@yasuking0304 I guess you missed this comment.

Reply:
If it was a person, I thought the direction of unification would be good, but when I checked, there are many other things and it is incomplete, so it seems that it should not be corrected.

Reply:
OK, so you replaced because you wanted to use only one of キャラクター and 人物 everywhere, i.e. for the consistency, right?
It sounds reasonable to replace the words.